tests/test_gnomad_vcf_to_protein.py

```diff
@@ -726,20 +726,26 @@ async def test_deletion(test_handler, protein_deletion_np_range,
 @pytest.mark.asyncio
 async def test_invalid(test_handler):
     """Test that invalid queries return correct response"""
-    resp = await test_handler.gnomad_vcf_to_protein("BRAF V600E")
+    resp = await test_handler.gnomad_vcf_to_protein("dummy")
+    assert resp.variation_descriptor is None
+
+    resp = await test_handler.gnomad_vcf_to_protein("BRAF V600E",
+                                                    untranslatable_returns_text=True)
```

Comment:
should we have a test case for `untranslatable_returns_text=False`?

Reply:
Default has `untranslatable_returns_text=False`

Reply:
(The example in the above lines tests this)